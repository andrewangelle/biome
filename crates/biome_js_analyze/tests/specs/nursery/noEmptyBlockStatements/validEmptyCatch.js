/* should not generate diagnostics */
try {
  doSomething();
} catch(ex) {}

function foo(){
  try {
      doSomething();
  } catch(ex) {}
}

const bar = () => {
  try {
      doSomething();
  } catch(ex) {}
}