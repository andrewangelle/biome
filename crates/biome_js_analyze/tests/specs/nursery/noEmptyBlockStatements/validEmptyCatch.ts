/* should not generate diagnostics */
const doSomethingTsValid = () => null;
try {
  doSomethingTsValid();
} catch(ex) {}

function fooTsValid(){
  try {
    doSomethingTsValid();
  } catch(ex) {}
}

const barTsValid = () => {
  try {
    doSomethingTsValid();
  } catch(ex) {}
}