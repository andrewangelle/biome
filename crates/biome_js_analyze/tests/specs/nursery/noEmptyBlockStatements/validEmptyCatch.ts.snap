---
source: crates/biome_js_analyze/tests/spec_tests.rs
expression: validEmptyCatch.ts
---
# Input
```js
/* should not generate diagnostics */
const doSomethingTsValid = () => null;
try {
  doSomethingTsValid();
} catch(ex) {}

function fooTsValid(){
  try {
    doSomethingTsValid();
  } catch(ex) {}
}

const barTsValid = () => {
  try {
    doSomethingTsValid();
  } catch(ex) {}
}
```


